src/get-started/java-binding.md

```diff
@@ -240,6 +240,33 @@ at `/cryptocurrency/balance/:walletId`.
 See [documentation][vertx.io] on the possibilities of `Vert.x` used as a web
 framework.
 
+### Supported Services
```

Comment:
this section locates inside the "Service Development" section. I'm not sure it's a good place. 
In my way, the place below "How to Run a Service" will be better.

Reply:
Also I'd clarify in the section name that those aren't just any services, but built-in services.